client/src/app/users/user.component.ts

```diff
@@ -1,37 +1,14 @@
-import {Component, OnInit} from '@angular/core';
-import {UserListService} from './user-list.service';
+import {Component, Input} from '@angular/core';
 import {User} from './user';
 
 @Component({
   selector: 'app-user-component',
   styleUrls: ['./user.component.css'],
   templateUrl: 'user.component.html'
 })
-export class UserComponent implements OnInit {
-  public user: User = null;
-  private id: string;
+export class UserComponent {
 
-  constructor(private userListService: UserListService) {
-    // this.users = this.userListService.getUsers();
-  }
+  @Input() user: User;
```

Comment:
@kklamberty  Currently there is only an input of 'user' in 'user.component.ts' and there is no function to test. What functionality should we add?